### ISSUE TITLE
Any way to build without clickable?

### ISSUE THREAD

**Issue by Crsarmv7l** [TronFourtyTwo/qvbam#2]
I am looking to build your app for the pinephone running arch and compiling directly on the pinephone. Looking through the clickable docs it seems it downloads and builds in a docker image which would be pretty hefty for the pinephone.

Are there any alt ways to build it? Thank you

**Comment by TronFortyTwo** [comment]
Yes. I did in the past. You can just build as a standard cmake project. You may need to compile the SDL library before,
